enhancements/cluster-logging/forward_to_cloudwatch.md

```diff
@@ -0,0 +1,198 @@
+---
+title: forward_to_cloudwatch
+authors:
+  - "@alanconway"
+reviewers:
+  - "@jcantrill"
+  - "@jeremyeder"
+approvers:
+creation-date: 2020-12-17
+last-updated: 2020-12-17
+status: implementable
+see-also:
+superseded-by:
+---
+
+# Forward to Cloudwatch
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+[Amazon Cloudwatch][aws-cw] is a hosted monitoring and log storage service.
+This proposal extends the `ClusterLogForwarder` API with an output type for Cloudwatch.
+
+## Motivation
+
+Amazon Cloudwatch is a popular log store.
+We have requests from external and  Red Hat-internal customers to support it.
+
+### Goals
+
+Enable log forwarding to Cloudwatch.
+
+### Non-Goals
+
+Enable Cloudwatch metric collection.
+
+## Proposal
+
+New API fields in the `output.cloudwatch` section:
+
+- `region`: (string) AWS region name.
+- `retentionDays`: (number) Number of days to keep logs. Has a default.
+- `group`: (string, *nice-to-have*) Cloudwatch log group name.
+   Defaults to the log type (application, infrastructure or audit)
```

Comment:
The "group" name in CW defaults to the log type?  Would expect this to default to something unique to the cluster such as infrastructures .status.infrastructureName

Reply:
I would ask the same question why the group would be used used to separate logs in lieu of logStream within a group?

Reply:
Need to check if region is a required parameter.
In the updated proposal the default is infra/app/audit, the user can also enable use of namespace for app logs. We don't have any named sub-categories of infra logs.
When you say "unique to the cluster" do you mean we need to prefix with a cluster-id?